src/components/templates/PartyMap/RoomPage/RoomPage.tsx

```diff
@@ -47,7 +46,7 @@ export default function RoomPage() {
     room.events && getCurrentEvent(room, venue.start_utc_seconds);
 
   return (
-    <WithNavigationBar redirectionUrl={`/v/${venue.id}`}>
+    <Modal show={true}>
```

Comment:
Placing the Modal in this file is fine for now, however in the future we'd probably want the parent component to decide (and therefore wrap) this component in a Modal or WithNavigationBar. For example what happens if someone shares a link to a friend directly to the room page?